package.json

```diff
@@ -74,9 +74,9 @@
     }
   },
   "dependencies": {
-    "@react-native-community/datetimepicker": "^3.4.7",
+    "@react-native-community/datetimepicker": "^3.5.0",
     "@react-native-community/masked-view": "^0.1.11",
-    "@react-native-community/picker": "^1.8.1",
+    "@react-native-community/picker": "1.8.1",
```

Comment:
Please do not fix the version on that package, let minor version update be used at package-lock.json.